rest_framework_inclusions/renderer.py

```diff
@@ -61,17 +62,30 @@ def _render_inclusions(self, data, renderer_context):
                     continue
                 render_data[key] = value
 
+        # map the meta information, if any
+        render_data[self.response_data_key] = serializer_data
+        render_data[self.response_inclusions_key] = inclusions
+
         return render_data
 
     def render(self, data, accepted_media_type=None, renderer_context=None):
         render_data = self._render_inclusions(data, renderer_context)
-        if not render_data:
+        allowed_actions = getattr(
+            renderer_context["view"], "include_allowed_actions", None
+        )
+        if not render_data or (
+            allowed_actions and renderer_context["view"].action not in allowed_actions
+        ):
             return super().render(data, accepted_media_type, renderer_context)
         return super().render(render_data, accepted_media_type, renderer_context)
 
 
-def get_allowed_paths(request):
-    include = request.GET.get("include") if request else None
+def get_allowed_paths(request, view):
+    if getattr(view, "get_include_allowed_paths", None):
+        include = view.get_include_allowed_paths(request)
```

Comment:
maybe `def get_requested_inclusions` as a method name that speaks a bit more to the imagination?